pkg/mtls/ca.go

```diff
@@ -193,7 +194,7 @@ func isClientCertificateSigned(PeerCertificates []*x509.Certificate, CAChain []*
 // registration endpoint: Any cert signed, even if it's expired.
 // All endpoints: checking that it's valid certificate.
 // @TODO check here the list of rejected certificates.
-func VerifyRequest(r *http.Request, verifyType int, verifyOpts x509.VerifyOptions, CACertChain []*x509.Certificate) bool {
+func VerifyRequest(r *http.Request, verifyType int, verifyOpts x509.VerifyOptions, CACertChain []*x509.Certificate, authzKey string) bool {
```

Comment:
Do we plan to use any other `authzKey`? ATM we always use `yggdrasil.AuthzKey`

Reply:
Just in case that we have a future RBAC with different key, owner for example.